### PR TITLE
ci: Fix cron schedule 

### PR DESCRIPTION
**Pull request by ko3n1g**
# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# GitHub Actions CI

The Jenkins CI system has been replaced by GitHub Actions self-hosted runners.

The GitHub Actions CI will run automatically when the "Run CICD" label is added to the PR.
To re-run CI remove and add the label again.
To run CI on an untrusted fork, a NeMo user with write access must first click "Approve and run".

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: .github/workflows/mcore-tag-bump-bot.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   main:
     runs-on: ubuntu-latest
-    environment: main
+    environment: ko3n1g/ci/improve-vm-maintenance
```
Comment (pablo-garay) [inline .github/workflows/mcore-tag-bump-bot.yml:11]: Please avoid using user name for Prod code

File: .github/workflows/monitor-vms.yml [hunk 2]
```diff
@@ -2,9 +2,8 @@
 name: Reboots VMs in a controlled way
 on:
   schedule:
-    - cron: /15 * * * *
```
Comment (pablo-garay) [inline .github/workflows/monitor-vms.yml:5]: Let's please be more cautious and test thoroughly properly before checking in as Prod code :)

File: .github/workflows/monitor-vms.yml [hunk 3]
```diff
@@ -28,7 +27,7 @@ jobs:
             | jq -c '[
                 .runners[] 
                 | select(.status == "online")
-                | select(.name | contains("gpu")
```
Comment (pablo-garay) [inline .github/workflows/monitor-vms.yml:31]: Let's please be more cautious and test thoroughly properly before checking in as Prod code :)

File: .github/workflows/mcore-tag-bump-bot.yml (hunk 1 above)
Comment (ko3n1g) [reply .github/workflows/mcore-tag-bump-bot.yml:11]: ups, thanks!

**Review by pablo-garay — APPROVED** [review]

**Pull request merged**